### DISCUSSION TITLE
Unable to import module 'lambda_handler': No module named 'pydantic_core._pydantic_core'

### DISCUSSION THREAD

**Discussion by evanerwee** [pydantic/pydantic#10190] [Question]
Since we upgraded to Pydatic v2 for our AWS Lambda workload, everything has been a mess. How do you get Pydantic v2 working in AWS?

We did upgrade our scripts: "pip install -r ../requirements.txt --platform manylinux_2_17_x86_64 --only-binary=:all: --target . --upgrade"

**Comment by evanerwee** [comment]
It appears, still doing testing, our script line should be: 
pip install -r ../requirements.txt --platform manylinux_2_17_x86_64 --only-binary=:all: --target . --upgrade --no-cache-dir

**Comment by VYalla** [comment]
Is there an update on this thread – did anyone find a solution? 

We're still running into getting Pydatic working on AWS Lambda
